### PR TITLE
change the Attackers paragraph in the atricle about Coincheck Hack

### PR DESCRIPTION
**Pull request by sln-dns**

### CODE REVIEW

**Review by eric-oconnor — COMMENTED** [review]
Regarding the 1st Submission: Poly Network Exploited, Largest DeFi Hack in Histor, there should be a y at the end of History

Regarding the 2nd Submission: Could you please add the relevant details regarding the hacker groups that are associated to this event and if there have been any additional updates.

Overall the submissions are decent, some quality assurance checks and clarifying details around these events should result in a successful submission.

**Comment by sln-dns** [comment]
> 


I could not find a source that made a definite judgment about the involvement of any group in this attack. 

The source would have to include an unambiguous identification of the attack, namely the date, the platform, the coins stolen, and the approximate amount. 

If you have any information or remember some facts but don't remember where you saw it, please let me know and I will make an effort to find information to update the article.

**Comment by eric-oconnor** [comment]
> > 
> 
> I could not find a source that made a definite judgment about the involvement of any group in this attack.
> 
> The source would have to include an unambiguous identification of the attack, namely the date, the platform, the coins stolen, and the approximate amount.
> 
> If you have any information or remember some facts but don't remember where you saw it, please let me know and I will make an effort to find information to update the article.

Correct in noting the lack of a definitive attribution, this is an important distinction to make while annotating data.  

While the attributions to a particular group are not definitive, they contain indispensable data and correlating attributes which are indispensable to be aware of when studying these attacks like vectors and methods, i.e., types of viruses or SE methods.  

Omitting information due to weak correlation keeps us from learning what is and isn't essential to be aware of in the future.

**Comment by sln-dns** [comment]
> 

Please check the new version of my article. I have changed the paragraph "Attackers"

[2019-01-28-Coincheck Hack.md](https://github.com/1712n/dni-website/files/11617689/2019-01-28-Coincheck.Hack.md)

**Comment by evgenydmitriev** [comment]
> > 
> 
> Please check the new version of my article. I have changed the paragraph "Attackers"
> 
> [2019-01-28-Coincheck Hack.md](https://github.com/1712n/dni-website/files/11617689/2019-01-28-Coincheck.Hack.md)

@sln-dns all your edits should be visible [here](https://github.com/1712n/dni-website/pull/28/files)

File: content/attacks/posts/2010-08-10-poly-network.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Shortly after the initial discovery, the hackers started engaging in a dialogue with the Poly Network team, expressing their intentions to return the stolen funds.
+- Over the following days, a complex negotiation process took place between the Poly Network team and the hackers, resulting in the gradual return of the majority of the stolen assets.
+- **August 15, 2021:** Poly Network issued a statement acknowledging the return of most of the stolen funds and expressed gratitude to the hacker(s) for their cooperation.
+- The incident prompted extensive analysis and discussions within the blockchain community regarding the vulnerabilities in DeFi protocols, smart contract audits, and the need for enhanced security measures.
+
+## Security Failure Causes
+
+The attack exploited a critical vulnerability in the Poly Network's smart contracts, enabling the unauthorized transfer of assets between different blockchain networks. The incident shed light on the challenges faced by DeFi projects in ensuring the robustness and resilience of their smart contracts, emphasizing the need for thorough security audits and ongoing vulnerability assessments.
+
+## Lessons Learned
```
Comment (evgenydmitriev) [inline content/attacks/posts/2010-08-10-poly-network.md:33]: don't invent sections. follow the structure outlined in the majority of the articles

**Comment by sln-dns** [comment]
Now I believe everything should be correct

File: content/attacks/posts/2010-08-10-poly-network.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Shortly after the initial discovery, the hackers started engaging in a dialogue with the Poly Network team, expressing their intentions to return the stolen funds.
+- Over the following days, a complex negotiation process took place between the Poly Network team and the hackers, resulting in the gradual return of the majority of the stolen assets.
+- **August 15, 2021:** Poly Network issued a statement acknowledging the return of most of the stolen funds and expressed gratitude to the hacker(s) for their cooperation.
+- The incident prompted extensive analysis and discussions within the blockchain community regarding the vulnerabilities in DeFi protocols, smart contract audits, and the need for enhanced security measures.
+
+## Security Failure Causes
+
+The attack exploited a critical vulnerability in the Poly Network's smart contracts, enabling the unauthorized transfer of assets between different blockchain networks. The incident shed light on the challenges faced by DeFi projects in ensuring the robustness and resilience of their smart contracts, emphasizing the need for thorough security audits and ongoing vulnerability assessments.
```
Comment (evgenydmitriev) [inline content/attacks/posts/2010-08-10-poly-network.md:31]: provide technical details, avoid general phrases about the importance of something

File: content/attacks/posts/2010-08-10-poly-network.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Attackers
+
+The identity of the attackers initially remained unknown. However, in a surprising turn of events, the hackers, acting under the pretext of exposing vulnerabilities and returning the stolen funds, engaged in a dialogue with the Poly Network team and eventually returned the majority of the stolen assets.
+
+## Losses
+
+The attack led to the theft of approximately $600 million worth of various cryptocurrencies, including Ethereum, Binance Smart Chain tokens, and Polygon tokens. The stolen assets comprised funds from different users, liquidity providers, and even the Poly Network treasury itself. The incident had a profound impact on the affected individuals and highlighted the financial risks associated with decentralized platforms.
```
Comment (evgenydmitriev) [inline content/attacks/posts/2010-08-10-poly-network.md:19]: avoid general phrases about the importance of something

File: content/attacks/posts/2010-08-10-poly-network.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Timeline
```
Comment (evgenydmitriev) [inline content/attacks/posts/2010-08-10-poly-network.md:21]: avoid bullet points without dates

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]
I reviewed only one article, but the same applies to the other one

**Comment by sln-dns** [comment]
> 
https://www.jcp.or.jp/akahata/aik17/2018-03-02/2018030201_03_1.html

　"Of the 523 million NEMs, the virtual currency that allegedly leaked from the virtual currency exchange operator Coincheck due to unauthorized access, at least 24 million had been transferred to Zaif, a major virtual currency exchange operator in Japan, by the end of February, this paper has learned."

**Comment by sln-dns** [comment]
> > 
> 
> https://www.jcp.or.jp/akahata/aik17/2018-03-02/2018030201_03_1.html
> 
> 　"Of the 523 million NEMs, the virtual currency that allegedly leaked from the virtual currency exchange operator Coincheck due to unauthorized access, at least 24 million had been transferred to Zaif, a major virtual currency exchange operator in Japan, by the end of February, this paper has learned."

It makes more sense to do one article-one pull request. 

However, I continue to do as Evgeny pointed out to me and put all the articles in one place. Maybe it is more convenient for you.

I also want to point out that the article "Poly Network" has already been accepted, nevertheless, I offered corrections for Evgeny's comments.

So at the moment, there are three articles:
1. Poly Network. Accepted but changed by Eugene's comments.
2. Coincheck Hack. Changed by Eugene's remarks.
3. Alpha Homora. Added a new one.

Thank you, it is interesting to work with you.

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]
Please pay attention to the [file naming instructions](https://github.com/1712n/dn-institute#:~:text=YYYY%2DMM%2DDD%2Dentity%2Dthat%2Dwas%2Dhacked.md).

**Comment by evgenydmitriev** [comment]
The pull request has been open for more than a week. Multiple conversation haven't been resolved. Closing.

@sln-dns if you want to submit any work in the future, try doing it in smaller chunks to make it more manageable and read up on GitHub pull request process.

**Pull request closed without merging**